content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -64,10 +65,11 @@ cluster for mesh expansion, run the following commands on a machine with cluster
     more about customizing Helm charts in the [Helm documentation](https://docs.helm.sh/using_helm/#using-helm).
     {{< /tip >}}
 
-1.  Find the IP address of the Istio ingress gateway, as this is how the mesh expansion machines will access [Citadel](/docs/concepts/security/) and [Pilot](/docs/concepts/traffic-management/#pilot-and-envoy).
+1.  Find the IP address of the Istio ingress gateway, as this is how the mesh expansion machines will access [Citadel](/docs/concepts/security/) and [Pilot](/docs/concepts/traffic-management/#pilot-and-envoy). Also throughout this doc, we assume the GCE VM is part of `$SERVICE_NAMESPACE`.
```

Comment:
This change isn't clear to me. At this point in the doc, the VM is not part of any namespace. Maybe you mean something like: ... we assume that the GCE VM will be added to the namespace `$SERVICE_NAMESPACE` ?

But that's also kind of weird since this is the first mention of SERVICE_NAMESPACE; how is a reader to know what is meant by SERVICE_NAMESPACE at this point in the doc?

Reply:
I add some sentence to explain. Feel free to suggest to expand it.